1001-2000/LIT1313Discou.xml

```diff
@@ -33,17 +44,47 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+        <abstract>
+<p>The text exists in two recensions: a) the older one, probably derived from Greek and transmitted as part of Sǝnodos (the present record) and
+b) the younger one, certainly translated from Arabic and transmitted as part of Haymānota ʾabaw.</p>
```

Comment:
if we know for sure the ID for the younger one will be LIT6819Teachi318A , I would refer already here to it (The younger one, `<ref type="work" corresp="LIT6819Teachi318A"/>`, ....)